src/StarcounterClientFiles/wwwroot/sys/app-shell/app-shell.html

```diff
@@ -3,7 +3,52 @@
 <head>
     <meta charset="utf-8">
     <title>{0}</title>
+    <script>  
+      (function() {{
+          const logPrefix = '[Starcounter App Shell]';
+          function register() {{
+          if ('serviceWorker' in navigator) {{
+            window.addEventListener('load', () => {{
+              const swUrl = `/service-worker.js`;
+              registerSW(swUrl);
+            }});
+          }}
+        }}
 
+        async function registerSW(swUrl) {{
+          try {{
+            const registration = await navigator.serviceWorker.register(swUrl);
+            
+            // send a message to the SW to it logs the SW disclaimer
+            navigator.serviceWorker.controller.postMessage("Hi! I deeclare myself as a new client");
```

Comment:
Typo: `deeclare` -> `declare`

Also, this line fails for me in Chrome in Omni when loading the page for the first time:

![image](https://user-images.githubusercontent.com/566463/56731788-9de18c00-675b-11e9-9805-d7eb0319389a.png)